### PR TITLE
[Docs] PR for Docs for Backup/Restore V2

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-3898

Preview: https://deploy-preview-12522--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/back-up-restore-universes/

### CODE REVIEW

**Comment by lizayugabyte** [comment]
@vpatibandla-yb Could you, please, review the first draft of the backup/restore doc?
Note that I only described how to create a scheduled backup policy, but not the actual backup that would use this policy because I couldn't find this functionality.

**Comment by lizayugabyte** [comment]
@vpatibandla-yb Could you, please, review the first draft of the backup/restore doc?
Note that I only described how to create a scheduled backup policy, but not the actual backup that would use this policy because I couldn't find this functionality.

**Comment by lizayugabyte** [comment]
@vpatibandla-yb Could you, please, review the first draft of the backup/restore doc?
Note that I only described how to create a scheduled backup policy, but not the actual backup that would use this policy because I couldn't find this functionality.

CC: @nbhatia-yb

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md [hunk 1]
```diff
@@ -32,22 +32,26 @@ showAsideToc: true
...
+![Create Backup](/images/yp/create-backup-new-1.png)
 
-1. Navigate to your universe and select **Backups**.
+To view details information about an existing backup, click on it to open **Backup Details**.
 
-1. Click **Create Backup** to open the **Create Backup** dialog shown in the following illustration:
+The **Backups** page allows you to create new backups that start immediately, as follows: 
 
-    <br/><br/>
+- Select the time period for the backup and click **Backup now** to open the dialog shown in the following illustration:<br><br>
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md:43]: What is the time period that you are referring to here?

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md [hunk 2]
```diff
@@ -32,22 +32,26 @@ showAsideToc: true
...
 
-    ![Create Backup - YCQL](/images/yp/create-backup-ycql.png)<br><br>
+  ![Backup](/images/yp/create-backup-new-3.png)<br><br>
 
-1. Complete the fields presented in the **YCQL** tab.
+- In the **Backup Now** dialog, select YCQL as the API type and then complete all the other fields.
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md:47]: In the case of YCQL, we should probably explain the flow to take a backup of a subset of tables in a keyspace. Will share the screenshots through slack.

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md [hunk 3]
```diff
@@ -32,22 +32,26 @@ showAsideToc: true
...
+  Notice that the contents of the **Select the storage config you want to use for your backup** field list depends on your existing backup storage configurations. For more information, see [Configure backup storage](../../configure-backup-storage/).
 
-1. Click **OK** to start the requested backup immediately.
+- Click **Backup**.
 
 If the universe has [encryption at rest enabled](../../../security/enable-encryption-at-rest), data files are backed up as-is (encrypted) to reduce the computation cost of a backup and to keep the files encrypted. A universe key metadata file containing key references is also backed up.
+
+For information on how to schedule backups for a later time or as a recurring task, see [Schedule universe YCQL data backups](../../schedule-data-backups/ycql/).
+
+To access a list of all backups from all universes, navigate to **Backups** on the YugabyteDB Anywhere left-side menu.
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md:57]: We should be explicitly stating that here we can see backups of deleted universes too

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md [hunk 4]
```diff
@@ -32,22 +32,26 @@ showAsideToc: true
...
 
-    Notice that the contents of the **Storage** field list depends on your existing backup storage configurations.
+  Notice that the contents of the **Select the storage config you want to use for your backup** field list depends on your existing backup storage configurations. For more information, see [Configure backup storage](../../configure-backup-storage/).
 
-1. Click **OK** to start the requested backup immediately.
+- Click **Backup**.
 
 If the universe has [encryption at rest enabled](../../../security/enable-encryption-at-rest), data files are backed up as-is (encrypted) to reduce the computation cost of a backup and to keep the files encrypted. A universe key metadata file containing key references is also backed up.
+
+For information on how to schedule backups for a later time or as a recurring task, see [Schedule universe YCQL data backups](../../schedule-data-backups/ycql/).
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md:55]: The schedule backup policy UI has changed and we should be providing new screenshots of the schedule policy UI

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ysql.md [hunk 5]
```diff
@@ -37,23 +37,23 @@ showAsideToc: true
 
 You can use YugabyteDB Anywhere to back up your YugabyteDB universe YSQL data.
 
-To schedule backups for a later time or as a recurring task, see [Schedule universe YSQL data backups](../../schedule-data-backups/ysql/).
+To view, [restore](../../restore-universe-data/ysql/), or delete existing backups for your universe, navigate to that universe and select **Backups**, as per the following illustration:
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ysql.md:40]: All the comments except the non-transactional one apply here too. There was never non-transactional backup support for YSQL.

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-universe-data/ycql.md [hunk 6]
```diff
@@ -30,29 +30,68 @@ showAsideToc: true
...
 </ul>
 
+## Basic restore procedure
+
 You can restore YugabyteDB universe YCQL data from a backup as follows:
 
-1. Open your universe and then select **Backups**.
+1. Open your universe and select **Backups**.
+
+2. In the **Backups** page, select the backup you want to restore and click its **... > Restore Entire Backup**, as per the following illustration:<br><br>
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-universe-data/ycql.md:39]: When we click on the backup, we list down all the keyspaces that were backed up. We can also select a keyspace from this list for restoring.

Comment (vpatibandla-yb) [reply docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-universe-data/ycql.md:39]: All the steps mentioned for the entire backup restore are applicable here too.

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-universe-data/ycql.md [hunk 7]
```diff
@@ -30,29 +30,68 @@ showAsideToc: true
...
+
+To proceed, complete the fields of the **Advanced Restore** dialog shown in the following illustration:
+
+![Restore advanced](/images/yp/restore-advanced-ycql.png)<br>
+
+- Select YCQL as the type of API.
+
+- Specify the location of the backup you want to restore.
+
+- Select the cloud provider-specific configuration of the backup storage. The storage could be on Google Cloud, Amazon S3, or Network File System.
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-universe-data/ycql.md:82]: Azure too

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-universe-data/ysql.md [hunk 8]
```diff
@@ -33,29 +33,64 @@ showAsideToc: true
...
 </ul>
 
+## Basic restore procedure
+
 You can restore YugabyteDB universe YSQL data from a backup as follows:
 
 1. Open your universe and select **Backups**.
 
-2. Click **Restore Backup** to open the **Restore data to** dialog shown in the following illustration:
+2. In the **Backups** page, select the backup you want to restore and click its **... > Restore Entire Backup**, as per the following illustration:<br><br>
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-universe-data/ysql.md:42]: The comment on ycql is valid here also

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/schedule-data-backups/ycql.md [hunk 9]
```diff
@@ -34,44 +34,39 @@ You can use YugabyteDB Anywhere to perform regularly scheduled backups of Yugaby
...
-
-    The initial backup begins immediately.
-
-Subsequent backups are created based on the value you specified for **Backup frequency** or **Cron expression**.
+Subsequent backups are created based on the value you specified for **Set backup intervals** or **Use cron expression**.
 
 ## Disable scheduled backups
 
-You can temporarily disable all scheduled backups as follows:
+You can temporarily disable all scheduled backups, as follows:
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/schedule-data-backups/ycql.md:64]: This step disables all kinds of backups on the universe. There is way to enable/disable schedule policy on each schedule defined

**Review by vpatibandla-yb — CHANGES_REQUESTED** [review]
There are a bunch of comments for which I will share the screenshots on slack

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md:43]: I misunderstood the time period selection box. Fixed the text now.

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ysql.md [hunk 10]
```diff
@@ -37,23 +37,23 @@ showAsideToc: true
...
+![Create Backup](/images/yp/create-backup-new-1.png)
 
-1. Open your universe and select **Backups**.
+To view details information about an existing backup, click on it to open **Backup Details**.
 
-1. Click **Create Backup** to open the **Create Backup** dialog shown in the following illustration:
+The **Backups** page allows you to create new backups that start immediately, as follows: 
 
-    <br/><br/>
+- Select the time period for the backup and click **Backup now** to open the dialog shown in the following illustration:<br><br>
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ysql.md:48]: I misunderstood the time period selection box. Fixed the text now.

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md:57]: Added info

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ysql.md (hunk 5 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ysql.md:40]: Okay

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-universe-data/ycql.md (hunk 6 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-universe-data/ycql.md:39]: Added a step for a keyspace restore

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-universe-data/ysql.md (hunk 8 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-universe-data/ysql.md:42]: Added keyspace restore step

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md (hunk 4 above)
Comment (vpatibandla-yb) [reply docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md:71]: The portal hasn't been upgraded after removing the feature flag. You will be able to notice the difference if you turn on the backupV2 feature under /features

Comment (vpatibandla-yb) [reply docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md:71]: Never mind. I see the new screenshots. This is primarily a UI change

**Review by vpatibandla-yb — APPROVED** [review]
Looks good to me

**Pull request merged**